src/peft/peft_model.py

```diff
@@ -159,7 +160,9 @@ def save_pretrained(self, save_directory, safe_serialization=False, **kwargs):
             peft_config.inference_mode = inference_mode
 
     @classmethod
-    def from_pretrained(cls, model, model_id, adapter_name="default", is_trainable=False, **kwargs):
+    def from_pretrained(
+        cls, model, model_id, adapter_name="default", config: Optional[PeftConfig] = None, is_trainable=False, **kwargs
```

Comment:
```suggestion
        cls, model, model_id, adapter_name="default", is_trainable=False, config: Optional[PeftConfig] = None, **kwargs
```

Reply:
I will add the test accordingly.

Reply:
We should swap this for positional not to break change